spectacles/lookml.py

```diff
@@ -424,18 +460,22 @@ def build_project(
             ),
         )
 
-    for model in models:
-        model.explores = [
-            explore
-            for explore in model.explores
-            if is_selected(model.name, explore.name, filters)
-        ]
-
-        if include_dimensions:
-            for explore in model.explores:
-                explore.dimensions = build_dimensions(
-                    client, model.name, explore.name, ignore_hidden_fields
-                )
-
-    project = Project(name, [model for model in models if len(model.explores) > 0])
-    return project
+    # For the content validator, we need all explores.
+    if not include_all_explores:
```

Comment:
I think this would be slightly easier to read if we swapped the if statement to `if include_all_explores`.